### PR TITLE
Enhancement/#3 - Log if any packet loss is detected & alter the method name

### PR DESCRIPTION
**Pull request by bschaatsbergen**
This aims to solve #3 .

### CODE REVIEW

**Comment by bschaatsbergen** [comment]
I now see that it's doing a `Errrof()`, quickly updating that.

File: pkg/ping/ping.go [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
 import (
 	"github.com/go-ping/ping"
 	log "github.com/sirupsen/logrus"
 )
 
-// Ping function to ping a host and return a set of statistics.
-// Uses the github.com/go-ping/ping module.
-func Ping(Address string, PacketCount int) *ping.Statistics {
-
+// Ping a host and return a set of statistics.
```
Comment (tiborhercz) [inline pkg/ping/ping.go:8]:
```suggestion
// pingAddress Ping a host and return a set of statistics.
```

File: pkg/ping/ping.go [hunk 2]
```diff
@@ -34,10 +32,13 @@ func Ping(Address string, PacketCount int) *ping.Statistics {
...
 }
 
-// MeasureLatency Exposes the average round trip time in milliseconds resolved from the Ping function.
-func MeasureLatency(Address string, PacketCount int) {
+// MeasureRtt Measures the average round-trip time in milliseconds.
+func MeasureRtt(Address string, PacketCount int) {
+	stats := pingAddress(Address, PacketCount)
 
-	stats := Ping(Address, PacketCount)
+	if stats.PacketLoss == 0 {
```
Comment (tiborhercz) [inline pkg/ping/ping.go:39]: Should this not be if the PacketLoss is bigger then 0?

Comment (bschaatsbergen) [reply pkg/ping/ping.go:39]: Looks like I messed up with the rebase

**Review by tiborhercz — APPROVED** [review]

**Pull request merged**